config/env_sample.hjson

```diff
@@ -90,7 +90,7 @@
         "ROOT_PASSWORD": "student_dashboard_root_pw"
     },
     # Default Canvas Data id increment for course id, user id, etc
```

Comment:
maybe add a comment here to explain "1000000000000" is the value used in UDP synthetic dataset, vs 17700000000000000 is a specific value for each Canvas institution. 

Something like:
```
# Default Canvas Data id increment for course id, user id, etc
# for Unizin synthetic data, the value is 1000000000000
"CANVAS_DATA_ID_INCREMENT": <institution_canvas_id>,
```